src/semgrep_agent/targets.py

```diff
@@ -70,11 +76,11 @@ class TargetFileManager:
     """
 
     _base_path = attr.ib(type=Path)
-    _paths = attr.ib(type=List[Path])
+    _all_paths = attr.ib(type=List[Path])
```

Comment:
Suggestion:

Maybe `_input_paths` or something would make this clear that it's the user input to the tool?